src/TwitchLib.Communication/Clients/AClientBase.cs

```diff
@@ -198,8 +200,11 @@ public bool Send(string message)
             LOGGER?.TraceMethodCall(GetType());
             try
             {
-                SendIRC(message);
-                return true;
+                lock (LOCK)
```

Comment:
is it wise to do a "normal" object lock vs e.g. using a SpinLock or a SemaphoreSlim?